.github/workflows/ci.yml

```diff
@@ -170,6 +170,18 @@ jobs:
       - name: Enable corepack and pnpm
         run: corepack enable && corepack enable pnpm
 
+      - name: Tee-worker client-api install npm deps
+        working-directory: ./tee-worker/client-api
+        run: pnpm install
+
+      - name: Tee-worker client-api build
+        working-directory: ./tee-worker/client-api
+        run: pnpm run build
```

Comment:
This is a `fmt` step - do we need to `run build` them?

Reply:
The correct steps should be:

```
1. install deps for client-api

2. build client-api

3. install deps for ts-tests

4. run ts-tests
```

fixed!